resources/views/dinnerform/admin_includes/dinnerform-list.blade.php

```diff
@@ -52,6 +53,16 @@
                         <td>{{ $dinnerform->start->format('Y m-d H:i') }}</td>
                         <td>{{ $dinnerform->end->format('Y m-d H:i') }}</td>
                         <td>€{{ number_format($dinnerform->totalAmountWithDiscount(), 2) }}</td>
+                        @if($dinnerform->ordered_by!=null)
```

Comment:
By removing the whole <td> element the items are shifted to the left when there is no ordered_by value. Change this to an empty <td> instead.